iframe/src/mocks/responses/steps/TestGateway/steps.ts

```diff
@@ -1,60 +1,49 @@
 import { BASE_API } from "../../../constants";
+import possibleFormFieldsStep from "./possibleFomStepFields";
 
 const firstStep = {
-	type: "form",
-	humanName: "Your Details",
-	url: `${BASE_API}/transaction/TestGateway/email/WyJHWHVZZGVBb1B6SF9JcXJWQXh6R3ZRLS0iLDEwMCwiRVVSIiwiQlRDIiwiY3JlZGl0Q2FyZCJd`,
-	title: "Your details",
-	useHeading: true,
-	data: [
-		{
-			type: "string",
-			name: "cryptocurrencyAddress",
-			humanName: "Cryptocurrency wallet address",
-			placeholder: "e.g 3FZbgi29cpjq2GjdwV8eyHuJJnkLtktZc5",
-		},
-		{
-			type: "string",
-			name: "fullname",
-			humanName: "Full name",
-			placeholder: "e.g JohnDoe",
-		},
-		{
-			type: "integer",
-			name: "phoneCountryCode",
-			humanName: "Phone country code",
-		},
-		{
-			type: "integer",
-			name: "phoneNumber",
-			humanName: "Phone number",
-			placeholder: "654 56 84 56",
-		},
-	],
+  type: "form",
+  progress: 30,
+  humanName: "Your Details",
+  url: `${BASE_API}/GoTo/TestGateway/emailStep/WyJHWHVZZGVBb1B6SF9JcXJWQXh6R3ZRLS0iLDEwMCwiRVVSIiwiQlRDIiwiY3JlZGl0Q2FyZCJd`,
+  title: "Your details",
+  useHeading: true,
+  data: [
+    {
+      type: "string",
+      name: "cryptocurrencyAddress",
+      humanName: "Cryptocurrency wallet address",
+      placeholder: "e.g 3FZbgi29cpjq2GjdwV8eyHuJJnkLtktZc5",
+    },
+    {
+      type: "string",
+      name: "fullname",
+      humanName: "Full name",
+      placeholder: "e.g JohnDoe",
+    },
+    {
+      type: "integer",
+      name: "phoneCountryCode",
+      humanName: "Phone country code",
+    },
+    {
+      type: "integer",
+      name: "phoneNumber",
+      humanName: "Phone number",
+      placeholder: "654 56 84 56",
+    },
+  ],
 };
 
-const emailStep = {
-	type: "redirect",
-	url: `${BASE_API}/transaction/TestGateway/verifyEmail/key`,
-	data: [
-		{
-			type: "string",
-			name: "email",
-			humanName: "Email",
-			hint: "You can also find the payment info in your email.",
-		}
-	],
-};
-
-const nextStep: {[key:string]: any} = {
-	firstStep,
-	emailStep
+const nextStep: { [key: string]: any } = {
+  firstStep, 
+  completed: { type: "completed", progress: 99, trackingUrl: "/" }
```

Comment:
Why is `progress: 99` included here? The progress is no longer visible on the completion screen. Also why not make it 100?